R/io.R

```diff
@@ -103,6 +111,31 @@ read_civis.sql <- function(x, database = NULL, using = utils::read.csv,
   })
 }
 
+#' @describeIn read_civis Return run output values or file ids of a \code{civis_script} as a list.
+#' @param regex Regex of matching run output names.
+#' @details
+#' If \code{using = NULL}, \code{read_civis.civis_script} will return run output
+#' file ids, including ids of JsonValues.
+#' Otherwise all run outputs matching \code{regex} will be read into memory
+#' with \code{using}. JsonValues are always returned as values if \code{using} is not \code{NULL}.
+#' Results are always a list. If the script has no outputs, the results are an empty list of length 0.
```

Comment:
What do you think about an implementation where instead of returning file_ids, this requires `using`?  Then `using=NULL` would correspond to JsonValues.  If a user wants file_ids, then they could use `fetch_output_file_ids`.

Reply:
I'm open to suggestions or leaving it as is, but maybe worth another second to think about.

Reply:
Yeah! I think that's a clearer way to handle both cases. Especially since we're exporting `fetch_output_file_ids`.